### PR TITLE
Added T[] constructor to VectorList

### PR DESCRIPTION
**Pull request by marcusnaslund**
This is a more general solution.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Why is this so impossible? :( @thomasfanell

File: source/geometry/FloatConvexHull2D.ooc [hunk 1]
```diff
@@ -25,9 +25,7 @@ FloatConvexHull2D: class {
 			this computeHull()
 	}
 	init: func ~fromArray (array: FloatPoint2D[], computeHull := true) {
-		list := VectorList<FloatPoint2D> new(array length)
-		for (i in 0 .. array length)
-			list add(array[i])
+		list := VectorList<FloatPoint2D> new(array)
```
Comment (sebastianbaginski) [inline source/geometry/FloatConvexHull2D.ooc]: Are you sure the correct constructor is called, like ooc is not interpreting "array" as Pointer, converting to Int and using this
```ooc
init: func ~heap (capacity: Int, freeContent := true) {
 		this init(HeapVector<T> new(1 maximum(capacity), freeContent))
 	}
```
instead ?

**Comment by thomasfanell** [comment]
Seems to that `T[]` requires `T` to be a class, not a cover. :heart:

**Comment by marcusnaslund** [comment]
C++, you say?

**Pull request closed without merging**